lib/index.js

```diff
@@ -0,0 +1,266 @@
+/*!
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const assert = require('assert-plus');
+const base64url = require('base64url-universal');
+const bedrock = require('bedrock');
+const brPackageManager = require('bedrock-package-manager');
+const crypto = require('crypto');
+const database = require('bedrock-mongodb');
+const {promisify} = require('util');
+const {util: {BedrockError, uuid}} = bedrock;
+
+// load config defaults
+require('./config');
+
+const AES_KW_ALGORITHM = 'id-aes256-wrap';
+const AES_KW_RFC3394_IV = Buffer.from('A6A6A6A6A6A6A6A6', 'hex');
+
+bedrock.events.on('bedrock.init', () => {
+  brPackageManager.register({
+    // FIXME: is this the alias for this package? This is what is already in
+    // place in kms stack
+    alias: 'bedrock-ssm',
+    packageName: 'bedrock-ssm-mongodb',
+    // FIXME: this `type` value is also up for bikeshedding
+    type: 'bedrock-kms-store',
```

Comment:
Type seems ok to me ... other people?

cc: @davidlehn, @gannan08

Reply:
It's fine with me, thought of maybe using `bedrock-kms-storage` but that did not seem right.